packages/atoms/src/ListItem/ListItemText.tsx

```diff
@@ -0,0 +1,30 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Text } from '../Text';
+import { View } from 'react-native';
+import React from 'react';
+import { BaseListItem } from './listItemTypes';
+
+type ListItemTextProps = Pick<BaseListItem, 'title' | 'subtitle' | 'wrap'>;
+
+const listItemTextStyle = prepareNativeStyle(() => ({
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 2,
+}));
+
+export const ListItemText = ({ wrap = false, title, subtitle }: ListItemTextProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const numberOfLines = {
+        numberOfLines: !wrap ? 1 : 0,
+    };
```

Comment:
I'm a little bit confused as to what is this doing? 😃

Reply:
According to https://github.com/trezor/trezor-suite/pull/5482#discussion_r882676891

> Setting this property to 0 will result in unsetting this value, which means that no lines restriction will be applied [1]

[1] https://reactnative.dev/docs/text#numberoflines

Reply:
Yep. It was just confusing mapping wrap and number of lines. And also why create an object with name and propertyName that's exactly same and then destructure it? 

You could just have `const numberOfLines = !wrap ? 1 : 0` and than `numberOfLines={numberOfLines}` in the cmp, no?

Reply:
You are right. Fixed in https://github.com/trezor/trezor-suite/pull/5482/commits/4d9801df045199b5ff6687636ce3debe4e694165